src/jit/namedintrinsiclist.h

```diff
@@ -66,6 +66,20 @@ enum HWIntrinsicFlag : unsigned int
     // Copy Upper bits
     // some SIMD scalar intrinsics need the semantics of copying upper bits from the source operand
     HW_Flag_CopyUpperBits = 0x200,
+
+    // Invert integral operands
+    HW_Flag_InvertIntegralOperands = 0x400,
+
+    // Select instruction using argument type
+    HW_Flag_InstrByArgType = 0x800,
```

Comment:
The name is confusing. For some intrinsics, the **base type** is from arguments (first one?) rather than instruction.

Reply:
This is applied to a relatively large group of instructions which cannot be selected based only on return type information and selection based on argument type is needed. I have not invented any name of reasonable length which would be more clear than the above one and would be happy to change to anything better.

Reply:
> This is applied to a relatively large group of instructions

Yes, this new flag looks good to me. But the name can be improved. How about `HW_Flag_BaseTypeFromArg`?

Reply:
> How about HW_Flag_BaseTypeFromArg?

@fiigii  Much better than mine. I will update code with new name.